docs/integrations/sources/hugging-face-datasets.md

```diff
@@ -0,0 +1,27 @@
+# Hugging Face - Datasets
+Imports datasets from Hugging Face ([https://huggingface.co/datasets](https://huggingface.co/datasets))
+
```

Comment:
Very nice 🤗 !
Connector makes use of the `/rows` endpoints, it works with parquet converted files, maybe worth mentioning it in doc.
```suggestion
Only datasets with [Parquet exports](https://huggingface.co/docs/dataset-viewer/en/parquet) can be imported with this connector.
```